src/quote/getQuoteAndBuyCoverInputs.ts

```diff
@@ -55,7 +55,7 @@ function getQuoteAndBuyCoverInputs(
   coverAsset: CoverAsset,
   coverBuyerAddress: Address,
   slippage?: number,
-  ipfsCid?: string,
+  ipfsCid?: string, // deprecated - passing ipfsCid now returns an error
   nexusApiUrl?: string,
 ): Promise<GetQuoteApiResponse | ErrorApiResponse>;
```

Comment:
i kept the `ipfsCid` overload function signature on `getQuoteAndBuyCoverInputs`, but now I'm returning an error if they pass an ipfsCid.

questions:
* should we just completely drop the overload signature?
* since this is a breaking change, who else uses this function aside from our FE and Brava (Athena)?

@shark0der @mixplore

Reply:
our FE doesn't use this, it uses only the upload method directly - I guess it's just Brava and OpenCover afaik 🤔

Reply:
overload function signature dropped